shell/platform/windows/flutter_windows_winuwp.cc

```diff
@@ -39,6 +56,31 @@ FlutterDesktopViewControllerCreateFromCoreWindow(
   state->view->SetEngine(
       std::unique_ptr<flutter::FlutterWindowsEngine>(EngineFromHandle(engine)));
   state->view->CreateRenderSurface();
+
+  winrt::Windows::ApplicationModel::Activation::IActivatedEventArgs
+      arg_interface{nullptr};
+  winrt::copy_from_abi(arg_interface, args);
+
+  std::vector<std::string> engine_switches;
+  winrt::Windows::ApplicationModel::Activation::LaunchActivatedEventArgs launch{
+      nullptr};
+  switch (arg_interface.Kind()) {
+    case winrt::Windows::ApplicationModel::Activation::ActivationKind::Launch:
```

Comment:
Are there going to be other kinds handled? It seems odd to use a `switch` instead of an `if` for a single case.